### ISSUE TITLE
filterTaps causes links and buttons not to be clickable on keyboard

### ISSUE THREAD

**Issue by uni0ue** [pmndrs/use-gesture#530]
**Describe the bug**
When `filterTaps` is set to `true` focused links or buttons can't be clicked by keyboard anymore. This is an accessibility and usability issue as [interactive elements must be able to be activated using a keyboard](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#interactive_elements_must_be_able_to_be_activated_using_a_keyboard).

This issue might be related to https://github.com/pmndrs/use-gesture/issues/418

**Sandbox or Video**
I've forked an existing sandbox and set `filterTaps: true` (and `preventDefault: true`) with draggable cards containing links and buttons: https://codesandbox.io/s/usegesture-filtertaps-slider-ggl7oz

When `filterTaps` is set to `false` the links and buttons are clickable by keyboard again. 

**Steps to reproduce:**
* Hit tab to focus the buttons and links
* Hit enter to click them
* Expected result: buttons and links click event is fired
* Actual result: click event is not fired

**Information:**
 - Use Gesture version: v10.2.17
 - Device: MacBook Pro
 - OS: macOS 12.0.1
 - Browser: Chrome 104.0.5112.79

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [x] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

**Comment by dbismut** [comment]
Hi @uni0ue thanks for raising an issue. Here is a minimal repro sandbox for reference: https://codesandbox.io/s/exciting-thompson-10gidk

I'll see if a fix is possible.

**Comment by dbismut** [comment]
Should be fixed in 10.2.18!
